pygeo/parameterization/DVGeo.py

```diff
@@ -506,7 +506,7 @@ def addRefAxis(
                     "rot0axis": rot0axis,
                 }
             nAxis = len(curve.coef)
-        elif xFraction or yFraction or zFraction:
+        elif xFraction is not None or yFraction is not None or zFraction is not None:
```

Comment:
The checks are good, but there are other if checks also later.

Reply:
Done